src/apportioned-emissions/daily/daily-apportioned-emissions.controller.spec.ts

```diff
@@ -27,7 +28,11 @@ describe('-- Daily Apportioned Emissions Controller --', () => {
     const module = await Test.createTestingModule({
       imports: [LoggerModule, StreamingModule],
       controllers: [DailyApportionedEmissionsController],
-      providers: [DayUnitDataRepository, DailyApportionedEmissionsService],
+      providers: [
+        ConfigService,
```

Comment:
ConfigService actually isn't needed here. I remember I had originally added it to the hourly stuff while fighting to get unit tests to run but I just checked and everything runs fine without configService so this could be removed.